blog/content/post/2020-12-10-OSFC_2020_part2.md

```diff
@@ -0,0 +1,170 @@
+presented details of coreboot port for POWER9 will covering hostboot, skiboot
+and petitboot and how they fit into coreboot firmware model. We have briefly
+described use cases, the project's development roadmap, the next steps within
+[Dasharo](dasharo.com) project, seamless integration of Intel FSP or AMD AGESA
+with UEFI-compliant and legacy OS interface.
+[Contributions](https://opensource.3mdeb.com/projects/coreboot.html).
+
+* [Trusted Execution Environment on OpenBMC](https://vimeo.com/488146312) by [Vernon Mauery](https://twitter.com/vmauery), [Zhikui Ren](https://cfp.osfc.io/osfc2020/speaker/SVSWZA/)
+
+BMC firmware for Intel. Interested improvements for ipmi password, ARM
+security (TEE) for protecting the password, and initial secret and safe/trusted
+execution environment. Widely discussed hardware requirements, boot
+requirements, and other prerequisites that are necessary for secure storage on
+the BMC. Presenters showed also planned improvements and the next steps of the
+development.
+
+* [AMD's Journey towards Open Source OpenBMC firmware](https://vimeo.com/488132697) by [Supreeth Venkatesh](https://cfp.osfc.io/osfc2020/speaker/RSCYCX/)
+
+The talk went through the main features implemented in meta-amd for OpenBMC,
+enabling it on AMD customer reference boards and upstream support for AMD
+system interfaces. Very detailed and exciting journey. Kudos to
+[AMD](https://www.amd.com/en) for your work.
+
+* [Introducing open firmware development model for the Programmable Service Engine's in Intel Atom x6000E Series](https://vimeo.com/488148045) by [Loo Tung Lun](https://cfp.osfc.io/osfc2020/speaker/8DGVWR/)
+
+Session described Internal CPU based on ARM Cortex M7 core, introduced in
+Elkhart Lake. Mentioned iotg-fbu to support PSE development. Overall, the
+presenters showed how to develop and customize the PSE firmware for Intel
+platform, by explaining the mechanisms of PSE firmware and system boot firmware
+cooperation with the open source software tools for integration and development.
+
+* [Open Source firmware testing with ConTest](https://vimeo.com/488148599) by [Christian Walter](https://twitter.com/nablahero), [Andrea Barberio](@insomniacslk)
+
+The presenters described ConTest - a modular framework aimed at automating
+system testing workflows and building board-specific testing infrastructure on
+top of it. It is a interesting project co-organized by Facebook + 9elelments,
+presenting ConTest as a major framework for OCP (Scenario OCP Deltalake). Kudos
+for sharing!  
+
+* [pure open source on an AMD Zen](https://vimeo.com/488147337) by [Ronald g. Minnich](https://twitter.com/coreboot)
+
+Very Rust talk dedicated to the project X aimed at existing binary blobs from
+the x86 part of Zen CPUs. Ron presented oreboot on the AMD Fam17h. He discussed
+the main problem of the Open System Firmware requirement for buildable,
+installable, redistributable firmware, the approaches to it, and the next steps.
```

Comment:
@miczyg1 can you comment on the real usefulness of this?
@KBanecka we should somehow add https://old.reddit.com/r/Amd/comments/kfqni7/osfc_2020_open_source_firmware_conference_pure/ggmqd8j/ and what is behind link pointed in my comment.